pomodoro.sh

```diff
@@ -83,20 +94,32 @@ chiming_with_input () {
         duration=$SECONDS
         clear_line
         echo "Chiming duration: $((duration / 60)) min $((duration % 60)) sec"
-        if [[ $input = "q" ]] || [[ $input = "Q" ]]; then
-            echo
+        if [[ $input = "q" || $input = "Q" ]]; then
```

Comment:
NIT:
```suggestion
        if [[ ${input^^} = "Q" ]]; then
```
`${input^^}`: the 2 carats capitalizes the variable